### ISSUE TITLE
Problem with "Sh"

### ISSUE THREAD

**Issue by IvanRebroff** [dscripka/openWakeWord#59]
Great thing so far, thank's a lot!
I'm trying to set some german wake words, e. g. "Schatzi" which I've tried with "Shutzy". But I only get "Hutzy". So I've tried some englisch words like "Show", "She" and some others and they are also pronounced wrong. Some words like "Sheila" or "Shooter" does work. Strange to me...

**Comment by dscripka** [comment]
This is very strange behavior indeed, @IvanRebroff.

After some testing I traced the issue to the Piper text-to-speech model that openWakeWord uses to generate training data, and I identified a potential solution. But I'm not sure if that fix would cause other problems, so I created an issue in [that repo](https://github.com/rhasspy/piper-sample-generator/issues/2) to get some additional feedback.

I'm glad you are finding openWakeWord useful, and I'm very interested to see if your models for German wake words work well! Proper multilingual support is something I hope to have eventually, and while certainly very limited, using english TTS models might work in certain narrow cases like this.
